Disable examples for power (ppc64le) installs

jeffdyoung opened the pull request:
@sdodson 

This worked for my power cluster, or would you prefer this logic be in openshift_examples?

I will still put the json definitions here:
roles/openshift_examples/files/examples/ppc64le/v3.x/imagestreams
roles/openshift_examples/files/examples/ppc64le/v3.x/db-templates
roles/openshift_examples/files/examples/ppc64le/v3.x/quickstart-templates

sdodson commented:
/lgtm
Did you want to go ahead and add those in this PR or do that as a followup?

jeffdyoung commented:
@sdodson  I'll add them here, should have it done by morning.

vrutkovs commented on playbooks/openshift-master/private/additional_config.yml:
```diff
@@ -26,8 +26,11 @@
+  # We don't want to populate examples on a power cluster.
   - role: openshift_examples
-    when: openshift_install_examples | default(true) | bool
+    when: 
```
```
playbooks/openshift-master/private/additional_config.yml:
  31:10     error    trailing spaces  (trailing-spaces)
  32:7      error    wrong indentation: expected 4 but found 6  (indentation)
```

sdodson commented:
@jeffdyoung Are the db-templates and quickstart-templates actually different from x86_64 content, like are some of them missing due to missing image streams? Or are the differences only in the image streams themselves?

jeffdyoung commented:
@sdodson Yes, if the imagestream doesn't exist on power, then I removed the corresponding template (like redis). I also updated any references to old imagestreams (ie..remove postgres 9.6,9.5 add postgres 10 to the db-template) and added 'ppc64le' in the tags. 
`            "tags": "database,postgresql,ppc64le"
`

Initially we had renamed all of the imagestreams xxxx_ppc64le, but we've backed away from that.
ATM, we felt it best to keep this simple, and change as little as possible.

sdodson commented:
ansible changes lgtm, if you're done with this please squash things into one commit and we can merge

sdodson commented:
/lgtm
Remove WIP from the title when you're ready and the bot should merge it.
